lib/cmark.ex

```diff
@@ -56,31 +122,47 @@ defmodule Cmark do
       ["HTML is <p>list</p>", "HTML is <p>test</p>"]
 
   """
-  def to_html_each(data, callback) when is_list(data) do
-    parse_doc_list_each(data, callback)
+  def to_html_each(data, callback, options \\ [:default]) when is_list(data) do
+    parse_doc_list_each(data, callback, options)
   end
 
-  defp parse_doc_list(documents) do
+  defp parse_doc_list(documents, options) do
     documents
-    |> Enum.map(&Task.async(fn -> parse_doc(&1) end))
+    |> Enum.map(&Task.async(fn -> parse_doc(&1, options) end))
     |> Enum.map(&Task.await(&1))
   end
 
-  defp parse_doc_list(documents, callback) do
-    callback.(parse_doc_list(documents))
+  defp parse_doc_list(documents, callback, options) do
+    callback.(parse_doc_list(documents, options))
   end
 
-  defp parse_doc_list_each(documents, callback) do
+  defp parse_doc_list_each(documents, callback, options) do
     documents
-    |> Enum.map(&Task.async(fn -> parse_doc(&1, callback) end))
+    |> Enum.map(&Task.async(fn -> parse_doc(&1, callback, options) end))
     |> Enum.map(&Task.await(&1))
   end
 
-  defp parse_doc(document) do
-    Cmark.Nif.to_html(document)
+  defp parse_doc(document, options) do
+    Cmark.Nif.to_html(document, parse_options(options))
   end
 
-  defp parse_doc(document, callback) do
-    callback.(Cmark.Nif.to_html(document))
+  defp parse_doc(document, callback, options) do
+    callback.(Cmark.Nif.to_html(document, parse_options(options)))
+  end
+
+  defp parse_options(options) do
+    use Bitwise
+
+    flags = %{
```

Comment:
No good reason, just some elixir ignorance.  I wasn't sure if a module attribute would be documented or not and I figured it was a private map so it didn't need to be exposed.